README.md

```
@@ -0,0 +1,88 @@
+├── report              # 신고
+│
+├── admin               # 관리자 기능
+├── version             # 클라이언트 앱 버전 관리
+└── test                # QA 용 테스트 데이터 초기화
+```
+
+<br/>
+
+## Team
+
+| <img src="https://avatars.githubusercontent.com/u/146558936?v=4" width="130" height="130"> | <img src="https://avatars.githubusercontent.com/u/115551339?v=4" width="130" height="130"> |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)